TestMusicPlayer fails to load AL library on lwjgl2

When I run the TestMusicPlayer app from jme3-examples, I immediately see 2 exceptions:

```console
Feb 21, 2023 9:25:51 PM com.jme3.system.JmeDesktopSystem initialize
INFO: Running on jMonkeyEngine 3.7.0-SNAPSHOT
 * Branch: master
 * Git Hash: 0cb5fe7
 * Build Date: 2023-02-21
Feb 21, 2023 9:25:51 PM com.jme3.audio.openal.ALAudioRenderer initOpenAL
SEVERE: Failed to load audio library
java.lang.UnsatisfiedLinkError: no lwjgl64 in java.library.path: /usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
	at java.base/java.lang.ClassLoader.loadLibrary(ClassLoader.java:2429)
	at java.base/java.lang.Runtime.loadLibrary0(Runtime.java:818)
	at java.base/java.lang.System.loadLibrary(System.java:1989)
	at org.lwjgl.Sys$1.run(Sys.java:72)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)
	at org.lwjgl.Sys.doLoadLibrary(Sys.java:66)
	at org.lwjgl.Sys.loadLibrary(Sys.java:87)
	at org.lwjgl.Sys.<clinit>(Sys.java:117)
	at org.lwjgl.openal.AL.<clinit>(AL.java:59)
	at com.jme3.audio.lwjgl.LwjglALC.isCreated(LwjglALC.java:61)
	at com.jme3.audio.openal.ALAudioRenderer.initOpenAL(ALAudioRenderer.java:93)
	at com.jme3.audio.openal.ALAudioRenderer.initialize(ALAudioRenderer.java:229)
	at jme3test.audio.TestMusicPlayer.initAudioPlayer(TestMusicPlayer.java:66)
	at jme3test.audio.TestMusicPlayer.<init>(TestMusicPlayer.java:58)
	at jme3test.audio.TestMusicPlayer$9.run(TestMusicPlayer.java:279)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:741)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)

Exception in thread "AWT-EventQueue-0" java.lang.NoClassDefFoundError: Could not initialize class org.lwjgl.openal.AL
	at com.jme3.audio.lwjgl.LwjglALC.destroyALC(LwjglALC.java:56)
	at com.jme3.audio.openal.ALAudioRenderer.destroyOpenAL(ALAudioRenderer.java:187)
	at com.jme3.audio.openal.ALAudioRenderer.cleanup(ALAudioRenderer.java:290)
	at jme3test.audio.TestMusicPlayer.formWindowClosing(TestMusicPlayer.java:247)
	at jme3test.audio.TestMusicPlayer.access$000(TestMusicPlayer.java:46)
	at jme3test.audio.TestMusicPlayer$1.windowClosing(TestMusicPlayer.java:95)
	at java.desktop/java.awt.AWTEventMulticaster.windowClosing(AWTEventMulticaster.java:357)
	at java.desktop/java.awt.AWTEventMulticaster.windowClosing(AWTEventMulticaster.java:357)
	at java.desktop/java.awt.Window.processWindowEvent(Window.java:2085)
	at java.desktop/javax.swing.JFrame.processWindowEvent(JFrame.java:298)
	at java.desktop/java.awt.Window.processEvent(Window.java:2044)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5001)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2780)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4833)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:773)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:746)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:744)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:743)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

I cloned the "fix-issue-1963" branch and tried to run `TestMusicPlayer` using LWJGL v2.

I got the following crash:
```java
Feb 22, 2023 10:18:29 AM com.jme3.system.JmeDesktopSystem initialize
INFO: Running on jMonkeyEngine null-SNAPSHOT
 * Branch: fix-issue-1963
 * Git Hash: cf38bbf
 * Build Date: 2023-02-22
Feb 22, 2023 10:18:29 AM com.jme3.audio.lwjgl.LwjglALC createALC
WARNING: Caught an LWJGLException from AL.create(). Will retry after 100 msec, with 3 more retries remaining.

Feb 22, 2023 10:18:29 AM com.jme3.audio.lwjgl.LwjglALC createALC
WARNING: Caught an LWJGLException from AL.create(). Will retry after 100 msec, with 2 more retries remaining.

Feb 22, 2023 10:18:29 AM com.jme3.audio.lwjgl.LwjglALC createALC
WARNING: Caught an LWJGLException from AL.create(). Will retry after 100 msec, with 1 more retry remaining.

Feb 22, 2023 10:18:29 AM com.jme3.audio.lwjgl.LwjglALC createALC
WARNING: Caught an LWJGLException from AL.create(). Will retry after 100 msec, with 0 more retries remaining.

Exception in thread "AWT-EventQueue-0" java.lang.RuntimeException: org.lwjgl.LWJGLException: Could not locate OpenAL library.
	at com.jme3.audio.lwjgl.LwjglALC.createALC(LwjglALC.java:32)
	at com.jme3.audio.openal.ALAudioRenderer.initOpenAL(ALAudioRenderer.java:94)
	at com.jme3.audio.openal.ALAudioRenderer.initialize(ALAudioRenderer.java:229)
	at jme3test.audio.TestMusicPlayer.initAudioPlayer(TestMusicPlayer.java:81)
	at jme3test.audio.TestMusicPlayer.<init>(TestMusicPlayer.java:73)
	at jme3test.audio.TestMusicPlayer$9.run(TestMusicPlayer.java:294)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:741)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: org.lwjgl.LWJGLException: Could not locate OpenAL library.
	at org.lwjgl.openal.AL.create(AL.java:156)
	at org.lwjgl.openal.AL.create(AL.java:102)
	at org.lwjgl.openal.AL.create(AL.java:206)
	at com.jme3.audio.lwjgl.LwjglALC.createALC(LwjglALC.java:27)
	... 18 more

```

--- a/jme3-examples/src/main/java/jme3test/audio/TestMusicPlayer.java
+++ b/jme3-examples/src/main/java/jme3test/audio/TestMusicPlayer.java
@@ -62,6 +62,7 @@ public class TestMusicPlayer extends javax.swing.JFrame {
             // started but in this test we do not need a LwjglContext, so
             // we should handle loading natives ourselves.
             NativeLibraryLoader.loadNativeLibrary("lwjgl", true);
+            NativeLibraryLoader.loadNativeLibrary("openal", true);
         } catch (ClassNotFoundException exception) {
             // Not using lwjgl version 2
         }

--- a/jme3-examples/src/main/java/jme3test/audio/TestMusicPlayer.java
+++ b/jme3-examples/src/main/java/jme3test/audio/TestMusicPlayer.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2009-2021 jMonkeyEngine
+ * Copyright (c) 2009-2023 jMonkeyEngine
  * All rights reserved.
  *
  * Redistribution and use in source and binary forms, with or without
@@ -40,6 +40,8 @@ import com.jme3.audio.plugins.OGGLoader;
 import com.jme3.audio.plugins.WAVLoader;
 import com.jme3.system.AppSettings;
 import com.jme3.system.JmeSystem;
+import com.jme3.system.NativeLibraryLoader;
+
 import java.io.*;
 import javax.swing.JFileChooser;
 
@@ -51,6 +53,19 @@ public class TestMusicPlayer extends javax.swing.JFrame {
     private float musicLength = 0;
     private float curTime = 0;
     final private Listener listener = new Listener();
+
+    static {
+        try {
+            Class.forName("org.lwjgl.opengl.Display", false, ClassLoader.getSystemClassLoader());
+            // Using lwjgl version 2
+            // In case of lwjgl2, natives are loaded when LwjglContext is
+            // started but in this test we do not need a LwjglContext, so
+            // we should handle loading natives ourselves.
+            NativeLibraryLoader.loadNativeLibrary("lwjgl", true);
+        } catch (ClassNotFoundException exception) {
+            // Not using lwjgl version 2
+        }
+    }
 
     public TestMusicPlayer() {
         initComponents();

--- a/jme3-examples/src/main/java/jme3test/audio/TestMusicPlayer.java
+++ b/jme3-examples/src/main/java/jme3test/audio/TestMusicPlayer.java
@@ -55,17 +55,14 @@ public class TestMusicPlayer extends javax.swing.JFrame {
     final private Listener listener = new Listener();
 
     static {
-        try {
-            Class.forName("org.lwjgl.opengl.Display", false, ClassLoader.getSystemClassLoader());
-            // Using lwjgl version 2
-            // In case of lwjgl2, natives are loaded when LwjglContext is
-            // started but in this test we do not need a LwjglContext, so
-            // we should handle loading natives ourselves.
-            NativeLibraryLoader.loadNativeLibrary("lwjgl", true);
-            NativeLibraryLoader.loadNativeLibrary("openal", true);
-        } catch (ClassNotFoundException exception) {
-            // Not using lwjgl version 2
-        }
+        // Load lwjgl and openal natives if lwjgl 2 is in classpath.
+        //
+        // In case of lwjgl2, natives are loaded when LwjglContext is
+        // started but in this test we do not create a LwjglContext,
+        // so we should handle loading natives ourselves if running
+        // with lwjgl 2.
+        NativeLibraryLoader.loadNativeLibrary("lwjgl", false);
+        NativeLibraryLoader.loadNativeLibrary("openal", false);
     }
 
     public TestMusicPlayer() {

--- a/jme3-examples/src/main/java/jme3test/audio/TestMusicPlayer.java
+++ b/jme3-examples/src/main/java/jme3test/audio/TestMusicPlayer.java
@@ -55,10 +55,10 @@ public class TestMusicPlayer extends javax.swing.JFrame {
     final private Listener listener = new Listener();
 
     static {
-        // Load lwjgl and openal natives if lwjgl 2 is in classpath.
+        // Load lwjgl and openal natives if lwjgl version 2 is in classpath.
         //
-        // In case of lwjgl2, natives are loaded when LwjglContext is
-        // started but in this test we do not create a LwjglContext,
+        // In case of lwjgl 2, natives are loaded when LwjglContext is
+        // started, but in this test we do not create a LwjglContext,
         // so we should handle loading natives ourselves if running
         // with lwjgl 2.
         NativeLibraryLoader.loadNativeLibrary("lwjgl", false);

